spec/requests.cddl

```diff
@@ -66,13 +66,21 @@ query-content = {
 query-response = tagged<{
   status: "replied"
   reply: call-reply
+  node_signatures: [* node-signature-of-query-response]
   //
   status: "rejected"
   reject_code: unsigned
   reject_message: text
   ? error_code: text
+  node_signatures: [* node-signature-of-query-response]
 }>
 
+node-signature-of-query-response = {
+  timestamp: timestamp
```

Comment:
Does this timestamp have a particular meaning across different replicas? Is it related to the timestamp in the state tree that this request is based on? Or is there really not anything we can say here?

Reply:
It does not have a particular meaning (yet). It is simply the timestamp when the signature was produced to allow clients check for replayed responses from far in the past. If we support making queries at a specified height (in the future), then we could relate it to the timestamp in the certified state tree.